src/node.cc

```diff
@@ -3950,6 +3955,8 @@ static void CheckIfAllowedInEnv(const char* exe, bool is_env,
     "--no-warnings",
     "--napi-modules",
     "--expose-http2",
+    "--experimental-modules",
+    "--loader",
```

Comment:
Perhaps `--module-loader` would be more descriptive?

Reply:
@jasnell what might it be confused by?